CHANGELOG.md

```diff
@@ -127,6 +127,12 @@ which will stop push notifications registering (and potentially asking for permi
  * The `DataBinding`, `EventBinding` and `ExpressionBinding` class constructors no longer take a `NameTable` argument.
  * The `Name` and `This` expression classes has been removed. The UX compiler will now compile these as `Constant` expressions that contain the actual objects instead.
  * The `IContext` interface no longer contains the `NameTable` property.
+ * The `Fuse.IRaw` interface removed (now internal to the `Fuse.Reactive.JavaScript` package). Had no practical public use.
+ * The `Fuse.Reactive.ListMirror` class is no longer public. This was never intended to be public and has no practical public application.
+ * Added detailed docs for many of the interfaces in the `Fuse.Reactive` namespace.
+ * The `Fuse.Reactive.IWriteable` interface has changed (breaking!). The method signature is now `bool TrySetExclusive(object)` instead of `void SetExclusive(object)`. Unlikely to affect your code.
+ * `IObservable` and `IObservableArray` no longer push their initial value on `Subscribe`.
+
```

Comment:
This is in the wrong part of the changelog, this is now under the 1.3-section. Please rebase on master, and move this under the unreleased-header.

Reply:
@kusma Done